src/Randomizer.SMZ3.Tracking/Configuration/ConfigMergeKeyAttribute.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using Randomizer.Shared;
+
+namespace Randomizer.SMZ3.Tracking.Configuration
+{
+    /// <summary>
+    /// Specifies the item categories for an item type.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Property,
+        Inherited = false, AllowMultiple = false)]
+    public sealed class ConfigMergeKeyAttribute : Attribute
```

Comment:
I'm not sure the "Config" part is actually all that relevant here, so I'd just call this MergeKey instead of ConfigMergeKey.